tools/c7n_azure/c7n_azure/resources/cdncustomdomain.py

```diff
@@ -0,0 +1,61 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+from c7n_azure.provider import resources
+from c7n_azure.resources.arm import ChildArmResourceManager
+from c7n_azure.utils import ResourceIdParser
+
+
+@resources.register('cdncustomdomain')
+class CdnCustomDomain(ChildArmResourceManager):
+    """CDN Endpoint Resource
+
+    :example:
+
+    Returns all CDN endpoints with Https provisioning stated not-equal Enabled
```

Comment:
rephrase for simplicity,     Returns all CDN endpoints without Https provisioning.